cpanspec

```diff
@@ -1323,22 +1323,22 @@ $description
 END
 
     my $all_patch_args_same = 1;
-    my $common_patch_args = undef;
+    my $common_patch_args = '-p1';
```

Comment:
Seems that this variable can be removed if you move `$autosetup_args` here. not a big difference but why to have two variables for the same value (`$common_patch_args` is used only for a comparison in the following block)